### PR TITLE
Perception: Working Visualization

### PR DESCRIPTION
**Pull request by eugenegx**

### CODE REVIEW

File: modules/perception/obstacle/camera/visualizer/frame_content.cc [hunk 1]
```diff
@@ -34,25 +34,16 @@ FrameContent::~FrameContent() {}
...
-
 void FrameContent::set_camera_content(double timestamp,
                                       Eigen::Matrix4d pose_c2w,
                                       const std::vector<ObjectPtr>& objects,
                                       const CameraFrameSupplement& supplement) {
-  if (camera_caches_.find(DoubleToMapKey(timestamp)) != camera_caches_.end()) {
-    return;
-  }
+  auto key = DoubleToMapKey(timestamp);
+  if (camera_caches_.count(key)) return;
```
Comment (Capri2014) [inline modules/perception/obstacle/camera/visualizer/frame_content.cc]: Do you want to add debug log to this abnormal return for easier debug later? Similarly to later ones

Comment (eugenegx) [reply modules/perception/obstacle/camera/visualizer/frame_content.cc]: No problem. Will do later

File: modules/perception/obstacle/onboard/camera_process_subnode.h [hunk 2]
```diff
@@ -90,6 +90,8 @@ class CameraProcessSubnode : public Subnode {
   CameraSharedData* cam_shared_data_;
 
   // Calibration
+  int32_t image_height_ = 1080;
```
Comment (Capri2014) [inline modules/perception/obstacle/onboard/camera_process_subnode.h]: consider change this one to common or gflags maybe? I saw these two in different image related files.

Comment (eugenegx) [reply modules/perception/obstacle/onboard/camera_process_subnode.h]: Will read it from Calibration service, which is not used now

**Pull request merged**